src/main/java/net/mcreator/ui/dialogs/wysiwyg/ButtonDialog.java

```diff
@@ -100,8 +98,8 @@ public ButtonDialog(WYSIWYGEditor editor, @Nullable Button button) {
 			} else {
 				int idx = editor.components.indexOf(button);
 				editor.components.remove(button);
-				Button buttonNew = new Button(button.name, button.getX(), button.getY(), text, button.width, button.height,
-						eh.getSelectedProcedure(), displayCondition.getSelectedProcedure());
+				Button buttonNew = new Button(button.name, button.getX(), button.getY(), text, button.width,

```

Comment:
Those line changes are unnecessary. We run formatter with fixed set of rules from time to time so those changes just make PR harder to review

Reply:
I just ran <kbd>Reformat code</kbd> in this file to adjust my own changes, how can these rules differ from device to device? (Because IIRC, they are packed with the repository.)

Reply:
Code is properly not formatted yet here properly. It is just easier to review if you don't do this + code formatting we do on master branch will take care of this sooner or later.

This change does not correlate to PR and does not help maintainers with reviewing the PR